temporal/api/cloud/identity/v1/message.proto

```diff
@@ -9,22 +9,52 @@ option java_outer_classname = "MessageProto";
 option ruby_package = "Temporalio::Api::Cloud::Identity::V1";
 option csharp_namespace = "Temporalio.Api.Cloud.Identity.V1";
 
+import "temporal/api/cloud/common/v1/message.proto";
 import "google/protobuf/timestamp.proto";
 
 message AccountAccess {
     // The role on the account, should be one of [admin, developer, read]
     // admin - gives full access the account, including users and namespaces
     // developer - gives access to create namespaces on the account
     // read - gives read only access to the account
-    string role = 1;
+    // Deprecated: Use role field instead.
+    string role_deprecated = 1 [deprecated = true];
+    // The role on the account.
+    // temporal:enums:replaces=role_deprecated
+    Role role = 2;
+
+    enum Role {
```

Comment:
Does defining it this way prevent us from referencing these role enums anywhere else?

Reply:
Not really, you can still reference it from other places. The enum will be scoped into the message. 
For e.g. when you refer the role you will have to refer it like this: 
```
    temporal.api.cloud.identity.v1.AccountAccess.Role role = 6;
```